packages/cli/src/lingui-extract.ts

```diff
@@ -26,6 +26,7 @@ export default async function command(
   config: LinguiConfig,
   options: Partial<CliExtractOptions>
 ): Promise<boolean> {
+  // todo validate and delete
```

Comment:
delete

Reply:
Please remove this comment if it is outdated.